db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180401034441) do
+ActiveRecord::Schema.define(version: 20180422031009) do
```

Comment:
Heads up, before you make any changes to the `SlackUser` migration file, you will need to roll back these two migrations.

The way to do that with docker is [make db_rollback](https://github.com/OperationCode/operationcode_backend/blob/master/Makefile#L46-L48). Running this command will rollback the *most recent* migration.  So you will need to run it twice, in order to get to the `SlackUser` migration.

Then you'll make your changes in the migration files.  And then run `make db_migrate` to run them, and update the `schema`.

Reply:
I did the rollback 2x, changed the schema, then performed `db_migrate`.